plugins/catalog/src/components/AboutCard/AboutCard.tsx

```diff
@@ -110,6 +112,19 @@ export function AboutCard(props: AboutCardProps) {
   const entityMetadataEditUrl =
     entity.metadata.annotations?.[ANNOTATION_EDIT_URL];
 
+  let techdocsRef: CompoundEntityRef | undefined;
+
+  if (entity.metadata.annotations?.['backstage.io/techdocs-entity']) {
```

Comment:
should probably do `const otherEntity = entity.metadata.annotations?.[TECHDOCS_EXTERNAL_ANNOTATION]` to leverage the constant and only do it once

Reply:
done